token-swap/program/src/instruction.rs

```diff
@@ -42,7 +45,9 @@ pub enum SwapInstruction {
     ///   3. `[writable]` token_(A|B) Base Account to swap INTO.  Must be the SOURCE token.
     ///   4. `[writable]` token_(A|B) Base Account to swap FROM.  Must be the DESTINATION token.
     ///   5. `[writable]` token_(A|B) DESTINATION Account assigned to USER as the owner.
-    ///   6. '[]` Token program id
+    ///   6. `[writable]` Pool token mint, to generate trading fees
+    ///   7. `[writable]` Fee account, to receive trading fees
+    ///   8. '[]` Token program id
```

Comment:
`process_swap` actually expects this order: 6. Token program id; 7. Pool token mint; 8. Fee account
Either fixup docs or implementation. Fwiw, I kind of like Token program id last, but your call.

Reply:
Great catch -- @bartosz-lipinski asked for them at the end for his integration, but I think he ended up using the JS package that we include.  I do also prefer the token program id at the end.  @bartosz-lipinski do you think we could switch them back?